programs/drift-competitions/src/instructions/claim_winnings.rs

```diff
@@ -5,32 +5,42 @@ use drift::state::insurance_fund_stake::InsuranceFundStake;
 use drift::state::spot_market::SpotMarket;
 
 use super::constraints::*;
-use crate::state::{Competition, Competitor};
-use drift::state::user::UserStats;
 use crate::error::ErrorCode;
-use drift::validate;
 use crate::signer_seeds::get_competition_authority_seeds;
-use drift::program::Drift;
+use crate::state::{Competition, Competitor};
 use drift::cpi::accounts::TransferProtocolIfShares;
-use drift::math::safe_math::SafeMath;
 use drift::math::casting::Cast;
+use drift::math::safe_math::SafeMath;
+use drift::program::Drift;
+use drift::state::user::UserStats;
+use drift::validate;
 
 pub fn claim_winnings<'info>(
     ctx: Context<'_, '_, '_, 'info, ClaimWinnings<'info>>,
     n_shares: Option<u64>,
```

Comment:
i feel like this should be a u128? will avoid some casts

Reply:
would probably need to change a bit to get this (like the u64 unclaimed_winnings on the account)

is it worth it?

Reply:
this also wasnt part of this PR